src/backend/distributed/commands/trigger.c

```diff
@@ -111,19 +117,24 @@ GetTriggerNameById(Oid triggerId)
 													useIndex, NULL, scanKeyCount,
 													scanKey);
 
-	char *triggerName = NULL;
+	HeapTuple targetHeapTuple = NULL;
 
 	HeapTuple heapTuple = systable_getnext(scanDescriptor);
 	if (HeapTupleIsValid(heapTuple))
 	{
-		Form_pg_trigger triggerForm = (Form_pg_trigger) GETSTRUCT(heapTuple);
-		triggerName = pstrdup(NameStr(triggerForm->tgname));
+		targetHeapTuple = heap_copytuple(heapTuple);
 	}
 
 	systable_endscan(scanDescriptor);
 	heap_close(pgTrigger, NoLock);
 
-	return triggerName;
+	if (targetHeapTuple == NULL && missingOk == false)
```

Comment:
Can we change the usages of `missingOk == false` with `!missingOk`

Reply:
tbh, I can't see much benefit in doing that conversation, to me, it is more explicit to check if it is `false`. What do you think ?

Reply:
Yeah I am okay with both of them, I think this comes down to personal choice, so you can keep `= false` as well :)